### PR TITLE
information-schema: add tidb_hot_regions_history docs and update tidb_hot_regions

### PR DESCRIPTION
**Pull request by IcePigZDB**
Signed-off-by: IcePigZDB <icepigzdb@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
* Add `information-schema/information-schema-tidb-hot-regions-history.md` to describes the usage of the new table`INFORMATION_SCHEMA.TIDB_HOT_REGIONS_HISTORY`.
* Update the description of `INFORMATION_SCHEMA.TIDB_HOT_REGIONS` and add link to `INFORMATION_SCHEMA.TIDB_HOT_REGIONS_HISTORY`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: pingcap/docs-cn/pull/7164
- Other reference link(s):
    Relative docs:
    * Chinese vergion: pingcap/docs-cn/pull/7163
    * English vergion: pingcap/docs/pull/6627
    
    Feature tracking Issue: 
    * pingcap/tidb/pull/25281

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by IcePigZDB** [comment]
> Hi, @IcePigZDB, could you please do the following when you finalize the PR:
> 
> 1. Check the links and markdown format because two Jenkins check items have failed.
> 2. Fill "What is changed, added or deleted? (Required)".
> 3. Specify the version to which you want to merge the PR.
>    Thanks.

roger that, I will mention @you and change thir PR `Ready for review` after fix them.

**Comment by TomShawn** [comment]
@IcePigZDB Please help confirm whether this feature is v5.3 related. If yes, please add the v5.3 label. Thanks!

**Comment by IcePigZDB** [comment]
In newest commit, I update the description of `INFORMATION_SCHEMA.TIDB_HOT_REGIONS` and add link to `INFORMATION_SCHEMA.TIDB_HOT_REGIONS_HISTORY`.
* Relative discussion here https://github.com/pingcap/tidb/pull/30660

File: information-schema/information-schema-tidb-hot-regions.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ summary: Learn the `TIDB_HOT_REGIONS` information_schema table.
 
 # TIDB_HOT_REGIONS
 
-The `TIDB_HOT_REGIONS` table provides information about hotspot Regions.
+The `TIDB_HOT_REGIONS` table provides information about current hotspot Regions, history hotspot is available in `[TIDB_HOT_REGIONS_HISTORY](/information-schema/information-schema-tidb-hot-regions-history.md)`.
```
Comment (hfxsd) [inline information-schema/information-schema-tidb-hot-regions.md:8]:
```suggestion
The `TIDB_HOT_REGIONS` table provides information about the current hot Regions. For information about history hot regions, see `[TIDB_HOT_REGIONS_HISTORY](/information-schema/information-schema-tidb-hot-regions-history.md)`.
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
+---
+title: TIDB_HOT_REGIONS_HISTORY
+summary: Learn the `TIDB_HOT_REGIONS_HISTORY` information_schema tabel.
+---
+
+# TIDB_HOT_REGIONS_HISTORY
+
+The `TIDB_HOT_REGIONS_HISTORY` table provides information about historical hot regions that are periodically recorded locally by PD, the recorded interval is the value of [' hot-regions-write-interval '](/pd-configuration-file. dm #hot-regions-write-interval) (the default value is 10minutes), the period for reserving historical hotspot information is the value of [' hot-regions-reserve-days'](/pd-configuration-file.m #hot-regions-write-interval) (the default value is 7days). See [PD configuration file description](/pd-configuration-file.md#hot-regions-write-interval) for details.
```
Comment (hfxsd) [inline information-schema/information-schema-tidb-hot-regions-history.md:8]:
```suggestion
The `TIDB_HOT_REGIONS_HISTORY` table provides information about history hot regions that are periodically recorded locally by PD. You can specify the recorded interval by configuring [' hot-regions-write-interval '](/pd-configuration-file. dm #hot-regions-write-interval). The default value is 10minutes. You can specify the period for reserving history information about hot regions by configuring [' hot-regions-reserve-days'](/pd-configuration-file.m #hot-regions-write-interval). The default value is 7days. See [PD configuration file description](/pd-configuration-file.md#hot-regions-write-interval) for details.
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+* IS_LEADER: Whether the PEER is the LEADER.
+* TYPE: The type of the hot Region.
+* HOT_DEGREE: The hot degree of the Region.
+* FLOW_BYTES: The number of bytes written and read in the Region.
+* KEY_RATE: The number of keys written and read in the Region.
+* QUERY_RATE: The number of queries written and read in the Region.
+
+> **Note:**
+>
+> + `UPDATE_TIME`, `REGION_ID`, `STORE_ID`, `PEER_ID`, `IS_LEARNER`, `IS_LEADER` and `TYPE` fields are pushed down to the PD servers for execution. To reduce the overhead of using the table, you must specify the time range for the search, , and as many conditions as possible. For example,  `select * from tidb_hot_regions_history where store_id = 11 and update_time > '2020-05-18 20:40:00' and update_time < '2020-05-18 21:40:00' and type='write'`.
```
Comment (hfxsd) [inline information-schema/information-schema-tidb-hot-regions-history.md:62]:
```suggestion
> + `UPDATE_TIME`, `REGION_ID`, `STORE_ID`, `PEER_ID`, `IS_LEARNER`, `IS_LEADER` and `TYPE` fields are pushed down to the PD servers for execution. To reduce the overhead of using the table, you must specify the time range for the search,  and specify as many conditions as possible. For example,  `select * from tidb_hot_regions_history where store_id = 11 and update_time > '2020-05-18 20:40:00' and update_time < '2020-05-18 21:40:00' and type='write'`.
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+Here are some common use cases:
```
Comment (hfxsd) [inline information-schema/information-schema-tidb-hot-regions-history.md:64]:
```suggestion
The following are some scenarios:
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+* Query hotspot regions within a specified period of time:
```
Comment (hfxsd) [inline information-schema/information-schema-tidb-hot-regions-history.md:66]:
```suggestion
* Query hot Regions within a specified period of time. Replace update_time with your actual time.
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+
+  {{< copyable "sql" >}}
+
+  ```sql
+  SELECT * FROM INFORMATION_SCHEMA.TIDB_HOT_REGIONS_HISTORY WHERE update_time >'2021-08-18 21:40:00' and update_time <'2021-09-19 00:00:00';
+  ```
+
+  > **Note:**
+  >
+  > + `UPDATE_TIME` also supports unix timestamps. For example, `update_time >TIMESTAMP('2021-08-18 21:40:00')` or `update_time > FROM_UNIXTIME(1629294000.000)` .
```
Comment (hfxsd) [inline information-schema/information-schema-tidb-hot-regions-history.md:76]:
```suggestion
  > + `UPDATE_TIME` also supports Unix timestamps. For example, `update_time >TIMESTAMP('2021-08-18 21:40:00')` or `update_time > FROM_UNIXTIME(1629294000.000)` .
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+
+* Query hotspot regions of a table within a specified period of time:
+
+  {{< copyable "sql" >}}
+
+  ```SQL
+  SELECT * FROM INFORMATION_SCHEMA.TIDB_HOT_REGIONS_HISTORY WHERE update_time >'2021-08-18 21:40:00' and update_time <'2021-09-19 00:00:00' and TABLE_NAME = 'table_name';
+  ```
+
+* Query the distribution of hotspot regions within a specified period of time:
```
Comment (hfxsd) [inline information-schema/information-schema-tidb-hot-regions-history.md:86]:
```suggestion
* Query the distribution of hot Regions within a specified period of time. Replace update_time and table_name with your actual values. 
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+
+  {{< copyable "sql" >}}
+
+  ```sql
+  SELECT count(region_id) cnt, store_id FROM INFORMATION_SCHEMA.TIDB_HOT_REGIONS_HISTORY WHERE update_time >'2021-08-18 21:40:00' and update_time <'2021-09-19 00:00:00'  and table_name = 'table_name' GROUP BY STORE_ID ORDER BY cnt DESC;
```
Comment (hfxsd) [inline information-schema/information-schema-tidb-hot-regions-history.md:91]:
```suggestion
  SELECT count(region_id) cnt, store_id FROM INFORMATION_SCHEMA.TIDB_HOT_REGIONS_HISTORY WHERE update_time >'2021-08-18 21:40:00' and update_time <'2021-09-19 00:00:00' and table_name = 'table_name' GROUP BY STORE_ID ORDER BY cnt DESC;
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+
+* Query the distribution of hotspot leader regions within a specified period of time:
+
+  {{< copyable "sql" >}}
+
+  ```sql
+  SELECT count(region_id) cnt, store_id FROM INFORMATION_SCHEMA.TIDB_HOT_REGIONS_HISTORY WHERE update_time >'2021-08-18 21:40:00' and update_time <'2021-09-19 00:00:00'  and table_name = 'table_name' and is_leader=1 GROUP BY STORE_ID ORDER BY cnt DESC;
+  ```
+
+* Query the distribution of hotspot index regions within a specified period of time:
```
Comment (hfxsd) [inline information-schema/information-schema-tidb-hot-regions-history.md:102]:
```suggestion
* Query the distribution of hot Index Regions within a specified period of time. Replace update_time and table_name with your actual values. 
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+
+  {{< copyable "sql" >}}
+
+  ```sql
+  SELECT count(region_id) cnt, index_name, store_id FROM INFORMATION_SCHEMA.TIDB_HOT_REGIONS_HISTORY WHERE update_time >'2021-08-18 21:40:00' and update_time <'2021-09-19 00:00:00' and table_name = 'table_name' group by index_name, store_id order by index_name,cnt desc;
+  ```
+
+* Query the distribution of hotspot index leader regions within a specified period of time:
```
Comment (hfxsd) [inline information-schema/information-schema-tidb-hot-regions-history.md:110]:
```suggestion
* Query the distribution of hot Index Leader Regions within a specified period of time. Replace update_time and table_name with your actual values. 
```

**Comment by IcePigZDB** [comment]
@hfxsd , I add a commit to keep description of tidb_hot_regions consistent with tidb_hot_regions_history.

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 11]
```diff
@@ -0,0 +1,116 @@
+---
+title: TIDB_HOT_REGIONS_HISTORY
+summary: Learn the `TIDB_HOT_REGIONS_HISTORY` information_schema table.
+---
+
+# TIDB_HOT_REGIONS_HISTORY
+
+The `TIDB_HOT_REGIONS_HISTORY` table provides information about history hot regions that are periodically recorded locally by PD. You can specify the recorded interval by configuring [' hot-regions-write-interval '](/pd-configuration-file. dm #hot-regions-write-interval). The default value is 10minutes. You can specify the period for reserving history information about hot regions by configuring [' hot-regions-reserve-days'](/pd-configuration-file.m #hot-regions-write-interval). The default value is 7days. See [PD configuration file description](/pd-configuration-file.md#hot-regions-write-interval) for details.
```
Comment (hfxsd) [inline information-schema/information-schema-tidb-hot-regions-history.md:8]:
```suggestion
The `TIDB_HOT_REGIONS_HISTORY` table provides information about history hot regions that are periodically recorded locally by PD. You can specify the recorded interval by configuring [' hot-regions-write-interval '](/pd-configuration-file.md#hot-regions-write-interval). The default value is 10 minutes. You can specify the period for reserving history information about hot regions by configuring [' hot-regions-reserve-days'](/pd-configuration-file.md#hot-regions-write-interval). The default value is 7 days. See [PD configuration file description](/pd-configuration-file.md#hot-regions-write-interval) for details.
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 12]
```diff
@@ -0,0 +1,116 @@
...
+* IS_LEADER: Whether the PEER is the LEADER.
+* TYPE: The type of the hot Region.
+* HOT_DEGREE: The hot degree of the hot Region.
+* FLOW_BYTES: The number of bytes written and read in the Region.
+* KEY_RATE: The number of keys written and read in the Region.
+* QUERY_RATE: The number of queries written and read in the Region.
+
+> **Note:**
+>
+> + `UPDATE_TIME`, `REGION_ID`, `STORE_ID`, `PEER_ID`, `IS_LEARNER`, `IS_LEADER` and `TYPE` fields are pushed down to the PD servers for execution. To reduce the overhead of using the table, you must specify the time range for the search,  and specify as many conditions as possible. For example,  `select * from tidb_hot_regions_history where store_id = 11 and update_time > '2020-05-18 20:40:00' and update_time < '2020-05-18 21:40:00' and type='write'`.
```
Comment (hfxsd) [inline information-schema/information-schema-tidb-hot-regions-history.md:62]:
```suggestion
> + `UPDATE_TIME`, `REGION_ID`, `STORE_ID`, `PEER_ID`, `IS_LEARNER`, `IS_LEADER` and `TYPE` fields are pushed down to the PD servers for execution. To reduce the overhead of using the table, you must specify the time range for the search, and specify as many conditions as possible. For example, `select * from tidb_hot_regions_history where store_id = 11 and update_time > '2020-05-18 20:40:00' and update_time < '2020-05-18 21:40:00' and type='write'`.
```

**Review by hfxsd — APPROVED** [review]
LGTM

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 13]
```diff
@@ -0,0 +1,116 @@
...
+| FLOW_BYTES  | double       | YES  |      | NULL    |       |
+| KEY_RATE    | double       | YES  |      | NULL    |       |
+| QUERY_RATE  | double       | YES  |      | NULL    |       |
++-------------+--------------+------+------+---------+-------+
+16 rows in set (0.00 sec)
+```
+
+The fields in the `TIDB_HOT_REGIONS_HISTORY` table are described as follows:
+
+* UPDATE_TIME: Update time of the hot Region.
```
Comment (hfxsd) [inline information-schema/information-schema-tidb-hot-regions-history.md:43]:
```suggestion
* UPDATE_TIME: The update time of the hot Region.
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 14]
```diff
@@ -0,0 +1,116 @@
+---
+title: TIDB_HOT_REGIONS_HISTORY
+summary: Learn the `TIDB_HOT_REGIONS_HISTORY` information_schema table.
+---
+
+# TIDB_HOT_REGIONS_HISTORY
+
+The `TIDB_HOT_REGIONS_HISTORY` table provides information about history hot regions that are periodically recorded locally by PD. You can specify the recorded interval by configuring [' hot-regions-write-interval '](/pd-configuration-file.md#hot-regions-write-interval). The default value is 10 minutes. You can specify the period for reserving history information about hot regions by configuring [' hot-regions-reserve-days'](/pd-configuration-file.md#hot-regions-write-interval). The default value is 7 days. See [PD configuration file description](/pd-configuration-file.md#hot-regions-write-interval) for details.
```
Comment (IcePigZDB) [inline information-schema/information-schema-tidb-hot-regions-history.md:8]:
```suggestion
The `TIDB_HOT_REGIONS_HISTORY` table provides information about history hot regions that are periodically recorded locally by PD. You can specify the recorded interval by configuring [' hot-regions-write-interval '](/pd-configuration-file.md#hot-regions-write-interval). The default value is 10 minutes. You can specify the period for reserving history information about hot regions by configuring [' hot-regions-reserve-days'](/pd-configuration-file.md#hot-regions-reserved-days). The default value is 7 days. See [PD configuration file description](/pd-configuration-file.md#hot-regions-write-interval) for details.
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 15]
```diff
@@ -0,0 +1,116 @@
+---
+title: TIDB_HOT_REGIONS_HISTORY
+summary: Learn the `TIDB_HOT_REGIONS_HISTORY` information_schema table.
+---
+
+# TIDB_HOT_REGIONS_HISTORY
+
+The `TIDB_HOT_REGIONS_HISTORY` table provides information about history hot regions that are periodically recorded locally by PD. You can specify the recorded interval by configuring [' hot-regions-write-interval '](/pd-configuration-file.md#hot-regions-write-interval). The default value is 10 minutes. You can specify the period for reserving history information about hot regions by configuring [' hot-regions-reserve-days'](/pd-configuration-file.md#hot-regions-reserved-days). The default value is 7 days. See [PD configuration file description](/pd-configuration-file.md#hot-regions-write-interval) for details.
```
Comment (IcePigZDB) [inline information-schema/information-schema-tidb-hot-regions-history.md:8]:
```suggestion
The `TIDB_HOT_REGIONS_HISTORY` table provides information about history hot regions that are periodically recorded locally by PD. You can specify the recorded interval by configuring [' hot-regions-write-interval '](/pd-configuration-file.md#hot-regions-write-interval). The default value is 10 minutes. You can specify the period for reserving history information about hot regions by configuring [' hot-regions-reserved-days'](/pd-configuration-file.md#hot-regions-reserved-days). The default value is 7 days. See [PD configuration file description](/pd-configuration-file.md#hot-regions-write-interval) for details.
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 16]
```diff
@@ -0,0 +1,116 @@
+---
+title: TIDB_HOT_REGIONS_HISTORY
+summary: Learn the `TIDB_HOT_REGIONS_HISTORY` information_schema table.
+---
+
+# TIDB_HOT_REGIONS_HISTORY
+
+The `TIDB_HOT_REGIONS_HISTORY` table provides information about history hot regions that are periodically recorded locally by PD. You can specify the recorded interval by configuring [' hot-regions-write-interval '](/pd-configuration-file.md#hot-regions-write-interval). The default value is 10 minutes. You can specify the period for reserving history information about hot regions by configuring [' hot-regions-reserved-days'](/pd-configuration-file.md#hot-regions-reserved-days). The default value is 7 days. See [PD configuration file description](/pd-configuration-file.md#hot-regions-write-interval) for details.
```
Comment (IcePigZDB) [inline information-schema/information-schema-tidb-hot-regions-history.md:8]:
```suggestion
The `TIDB_HOT_REGIONS_HISTORY` table provides information about history hot regions that are periodically recorded locally by PD. You can specify the recorded interval by configuring ['hot-regions-write-interval '](/pd-configuration-file.md#hot-regions-write-interval). The default value is 10 minutes. You can specify the period for reserving history information about hot regions by configuring ['hot-regions-reserved-days'](/pd-configuration-file.md#hot-regions-reserved-days). The default value is 7 days. See [PD configuration file description](/pd-configuration-file.md#hot-regions-write-interval) for details.
```

Comment (IcePigZDB) [inline information-schema/information-schema-tidb-hot-regions-history.md:8]:
```suggestion
The `TIDB_HOT_REGIONS_HISTORY` table provides information about history hot regions that are periodically recorded locally by PD. You can specify the recorded interval by configuring ['hot-regions-write-interval'](/pd-configuration-file.md#hot-regions-write-interval). The default value is 10 minutes. You can specify the period for reserving history information about hot regions by configuring ['hot-regions-reserved-days'](/pd-configuration-file.md#hot-regions-reserved-days). The default value is 7 days. See [PD configuration file description](/pd-configuration-file.md#hot-regions-write-interval) for details.
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 17]
```diff
@@ -0,0 +1,116 @@
+---
+title: TIDB_HOT_REGIONS_HISTORY
+summary: Learn the `TIDB_HOT_REGIONS_HISTORY` information_schema table.
+---
+
+# TIDB_HOT_REGIONS_HISTORY
+
+The `TIDB_HOT_REGIONS_HISTORY` table provides information about history hot regions that are periodically recorded locally by PD. You can specify the recorded interval by configuring ['hot-regions-write-interval'](/pd-configuration-file.md#hot-regions-write-interval). The default value is 10 minutes. You can specify the period for reserving history information about hot regions by configuring ['hot-regions-reserved-days'](/pd-configuration-file.md#hot-regions-reserved-days). The default value is 7 days. See [PD configuration file description](/pd-configuration-file.md#hot-regions-write-interval) for details.
```
Comment (TomShawn) [inline information-schema/information-schema-tidb-hot-regions-history.md:8]:
```suggestion
The `TIDB_HOT_REGIONS_HISTORY` table provides information about history hot Regions that are periodically recorded locally by PD. You can specify the record interval by configuring [`hot-regions-write-interval`](/pd-configuration-file.md#hot-regions-write-interval). The default value is 10 minutes. You can specify the period for reserving history information about hot Regions by configuring [`hot-regions-reserved-days`](/pd-configuration-file.md#hot-regions-reserved-days). The default value is 7 days. See [PD configuration file description](/pd-configuration-file.md#hot-regions-write-interval) for details.
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 18]
```diff
@@ -0,0 +1,116 @@
...
+* IS_LEADER: Whether the PEER is the LEADER.
+* TYPE: The type of the hot Region.
+* HOT_DEGREE: The hot degree of the hot Region.
+* FLOW_BYTES: The number of bytes written and read in the Region.
+* KEY_RATE: The number of keys written and read in the Region.
+* QUERY_RATE: The number of queries written and read in the Region.
+
+> **Note:**
+>
+> + `UPDATE_TIME`, `REGION_ID`, `STORE_ID`, `PEER_ID`, `IS_LEARNER`, `IS_LEADER` and `TYPE` fields are pushed down to the PD servers for execution. To reduce the overhead of using the table, you must specify the time range for the search, and specify as many conditions as possible. For example, `select * from tidb_hot_regions_history where store_id = 11 and update_time > '2020-05-18 20:40:00' and update_time < '2020-05-18 21:40:00' and type='write'`.
```
Comment (TomShawn) [inline information-schema/information-schema-tidb-hot-regions-history.md:62]:
```suggestion
> `UPDATE_TIME`, `REGION_ID`, `STORE_ID`, `PEER_ID`, `IS_LEARNER`, `IS_LEADER` and `TYPE` fields are pushed down to the PD servers for execution. To reduce the overhead of using the table, you must specify the time range for the search, and specify as many conditions as possible. For example, `select * from tidb_hot_regions_history where store_id = 11 and update_time > '2020-05-18 20:40:00' and update_time < '2020-05-18 21:40:00' and type='write'`.
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 19]
```diff
@@ -0,0 +1,116 @@
...
+
+The following are some scenarios:
```
Comment (TomShawn) [inline information-schema/information-schema-tidb-hot-regions-history.md:64]:
```suggestion
Common user scenarios
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 20]
```diff
@@ -0,0 +1,116 @@
...
+
+* Query hot Regions within a specific period of time. Replace update_time with your actual time.
```
Comment (TomShawn) [inline information-schema/information-schema-tidb-hot-regions-history.md:66]:
```suggestion
* Query hot Regions within a specific period of time. Replace `update_time` with your actual time.
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 21]
```diff
@@ -0,0 +1,116 @@
...
+
+  {{< copyable "sql" >}}
+
+  ```sql
+  SELECT * FROM INFORMATION_SCHEMA.TIDB_HOT_REGIONS_HISTORY WHERE update_time >'2021-08-18 21:40:00' and update_time <'2021-09-19 00:00:00';
+  ```
+
+  > **Note:**
+  >
+  > + `UPDATE_TIME` also supports Unix timestamps. For example, `update_time >TIMESTAMP('2021-08-18 21:40:00')` or `update_time > FROM_UNIXTIME(1629294000.000)` .
```
Comment (TomShawn) [inline information-schema/information-schema-tidb-hot-regions-history.md:76]:
```suggestion
  > `UPDATE_TIME` also supports Unix timestamps. For example, `update_time >TIMESTAMP('2021-08-18 21:40:00')` or `update_time > FROM_UNIXTIME(1629294000.000)` .
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 22]
```diff
@@ -0,0 +1,116 @@
...
+
+* Query hot Regions in a table within a specific period of time. Replace update_time and table_name with your actual values.
```
Comment (TomShawn) [inline information-schema/information-schema-tidb-hot-regions-history.md:78]:
```suggestion
* Query hot Regions in a table within a specific period of time. Replace `update_time` and `table_name` with your actual values.
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 23]
```diff
@@ -0,0 +1,116 @@
...
+
+  {{< copyable "sql" >}}
+
+  ```SQL
+  SELECT * FROM INFORMATION_SCHEMA.TIDB_HOT_REGIONS_HISTORY WHERE update_time >'2021-08-18 21:40:00' and update_time <'2021-09-19 00:00:00' and TABLE_NAME = 'table_name';
+  ```
+
+* Query the distribution of hot Regions within a specific period of time. Replace update_time and table_name with your actual values.
```
Comment (TomShawn) [inline information-schema/information-schema-tidb-hot-regions-history.md:86]:
```suggestion
* Query the distribution of hot Regions within a specific period of time. Replace `update_time` and `table_name` with your actual values.
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 24]
```diff
@@ -0,0 +1,116 @@
...
+
+  {{< copyable "sql" >}}
+
+  ```sql
+  SELECT count(region_id) cnt, store_id FROM INFORMATION_SCHEMA.TIDB_HOT_REGIONS_HISTORY WHERE update_time >'2021-08-18 21:40:00' and update_time <'2021-09-19 00:00:00' and table_name = 'table_name' GROUP BY STORE_ID ORDER BY cnt DESC;
+  ```
+
+* Query the distribution of hot Leader Regions within a specific period of time. Replace update_time and table_name with your actual values.
```
Comment (TomShawn) [inline information-schema/information-schema-tidb-hot-regions-history.md:94]:
```suggestion
* Query the distribution of hot Leader Regions within a specific period of time. Replace `update_time` and `table_name` with your actual values.
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 25]
```diff
@@ -0,0 +1,116 @@
...
+
+  {{< copyable "sql" >}}
+
+  ```sql
+  SELECT count(region_id) cnt, store_id FROM INFORMATION_SCHEMA.TIDB_HOT_REGIONS_HISTORY WHERE update_time >'2021-08-18 21:40:00' and update_time <'2021-09-19 00:00:00' and table_name = 'table_name' and is_leader=1 GROUP BY STORE_ID ORDER BY cnt DESC;
+  ```
+
+* Query the distribution of hot Index Regions within a specific period of time. Replace update_time and table_name with your actual values.
```
Comment (TomShawn) [inline information-schema/information-schema-tidb-hot-regions-history.md:102]:
```suggestion
* Query the distribution of hot Index Regions within a specific period of time. Replace `update_time` and `table_name` with your actual values.
```

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 26]
```diff
@@ -0,0 +1,116 @@
...
+
+  {{< copyable "sql" >}}
+
+  ```sql
+  SELECT count(region_id) cnt, index_name, store_id FROM INFORMATION_SCHEMA.TIDB_HOT_REGIONS_HISTORY WHERE update_time >'2021-08-18 21:40:00' and update_time <'2021-09-19 00:00:00' and table_name = 'table_name' group by index_name, store_id order by index_name,cnt desc;
+  ```
+
+* Query the distribution of hot Index Leader Regions within a specific period of time. Replace update_time and table_name with your actual values.
```
Comment (TomShawn) [inline information-schema/information-schema-tidb-hot-regions-history.md:110]:
```suggestion
* Query the distribution of hot Index Leader Regions within a specific period of time. Replace `update_time` and `table_name` with your actual values.
```

File: information-schema/information-schema-tidb-hot-regions.md [hunk 27]
```diff
@@ -5,7 +5,7 @@ summary: Learn the `TIDB_HOT_REGIONS` information_schema table.
 
 # TIDB_HOT_REGIONS
 
-The `TIDB_HOT_REGIONS` table provides information about hotspot Regions.
+The `TIDB_HOT_REGIONS` table provides information about the current hot Regions. For information about history hot regions, see `[TIDB_HOT_REGIONS_HISTORY](/information-schema/information-schema-tidb-hot-regions-history.md)`.
```
Comment (TomShawn) [inline information-schema/information-schema-tidb-hot-regions.md:8]:
```suggestion
The `TIDB_HOT_REGIONS` table provides information about the current hot Regions. For information about history hot Regions, see `[TIDB_HOT_REGIONS_HISTORY](/information-schema/information-schema-tidb-hot-regions-history.md)`.
```

File: information-schema/information-schema.md [hunk 28]
```diff
@@ -83,6 +83,7 @@ Many `INFORMATION_SCHEMA` tables have a corresponding `SHOW` command. The benefi
 | [`STATEMENTS_SUMMARY_HISTORY`](/statement-summary-tables.md)                            | Similar to performance_schema statement summary history in MySQL. |
 | [`TABLE_STORAGE_STATS`](/information-schema/information-schema-table-storage-stats.md)  | Provides details about table sizes in storage. |
 | [`TIDB_HOT_REGIONS`](/information-schema/information-schema-tidb-hot-regions.md)        | Provides statistics about which regions are hot. |
+| [`TIDB_HOT_REGIONS_HISTORY`](/information-schema/information-schema-tidb-hot-regions-history.md) | Provides history statistics about which regions are hot. |
```
Comment (TomShawn) [inline information-schema/information-schema.md:86]:
```suggestion
| [`TIDB_HOT_REGIONS_HISTORY`](/information-schema/information-schema-tidb-hot-regions-history.md) | Provides history statistics about which Regions are hot. |
```

**Review by TomShawn — APPROVED** [review]
Minor comments

File: information-schema/information-schema-tidb-hot-regions-history.md [hunk 29]
```diff
@@ -0,0 +1,116 @@
+---
+title: TIDB_HOT_REGIONS_HISTORY
+summary: Learn the `TIDB_HOT_REGIONS_HISTORY` information_schema table.
+---
+
+# TIDB_HOT_REGIONS_HISTORY
+
+The `TIDB_HOT_REGIONS_HISTORY` table provides information about history hot Regions that are periodically recorded locally by PD. You can specify the record interval by configuring [`hot-regions-write-interval`](/pd-configuration-file.md#hot-regions-write-interval). The default value is 10 minutes. You can specify the period for reserving history information about hot Regions by configuring [`hot-regions-reserved-days`](/pd-configuration-file.md#hot-regions-reserved-days). The default value is 7 days. See [PD configuration file description](/pd-configuration-file.md#hot-regions-write-interval) for details.
```
Comment (TomShawn) [inline information-schema/information-schema-tidb-hot-regions-history.md:8]:
```suggestion
The `TIDB_HOT_REGIONS_HISTORY` table provides information about history hot Regions that are periodically recorded locally by PD. You can specify the record interval by configuring [`hot-regions-write-interval`](/pd-configuration-file.md#hot-regions-write-interval-new-in-v540). The default value is 10 minutes. You can specify the period for reserving history information about hot Regions by configuring [`hot-regions-reserved-days`](/pd-configuration-file.md#hot-regions-reserved-days-new-in-v540). The default value is 7 days. See [PD configuration file description](/pd-configuration-file.md#hot-regions-write-interval-new-in-v540) for details.
```

**Pull request merged**